engine/runtime/src/main/java/org/enso/interpreter/runtime/scope/ModuleScope.java

```diff
@@ -305,15 +352,18 @@ public Set<String> getMethodNamesForType(Type tpe) {
    */
   public void registerAllMethodsOfTypeToScope(Type tpe, ModuleScope scope) {
     Type tpeKey = tpe == null ? noTypeKey : tpe;
-    Map<String, Function> allTypeMethods = methods.get(tpeKey);
+    var allTypeMethods = methods.get(tpeKey);
```

Comment:
Explicit type signatures make the code easier to understand

Reply:
Certainly. On the other hand we also use `val` in Scala everywhere.